README.md

```diff
@@ -3,17 +3,17 @@
 
 ## Table of Contents
 - [Introduction](#introduction)
-- [Getting Started](#getting-started)
-- [Contributing](#contributing)
+- [End Users](#end-users)
```

Comment:
This is a link to the section in this README, not to the end_users section of lore, so no, I do not think this should be made to match the directory name.